addons/mrp/models/mrp_workorder.py

```diff
@@ -782,7 +785,11 @@ def _get_conflicted_workorder_ids(self):
         res = defaultdict(list)
         for wo1, wo2 in self.env.cr.fetchall():
             res[wo1].append(wo2)
-        return res
+        res_simultaneous_wo = defaultdict(list)
```

Comment:
The SQL means nothing anymore. Should be done on ressources directly. Check if two leaves for the same interval overlaps